added index, style, readme, and asset images

Chike1990 opened the pull request.

Chike1990 committed:
added style file

Chike1990 committed:
added index file

Chike1990 committed:
added .github folder

Chike1990 committed:
added a picture

Chike1990 committed:
Fix errors indicated on linters

Chike1990 committed:
Added and styled the first part of the footer; included an an image to the folder

Chike1990 committed:
Some more work on the footer

Chike1990 committed:
Positioning

Chike1990 committed:
Corrects linter issues

Chike1990 committed:
Styling CSS linters

Chike1990 committed:
fix index linters

Chike1990 committed:
fixes a little index issues

NtwaliHeritier reviewed (changes requested):
### **STATUS: CHANGES REQUESTED🛑🛑🛑🛑**
Hello @Chike1990 ,
Great work on the project
Let's make some changes to make it even more professional

[REQUIRED]: Update your readme file to show link the live demo and how to set up the project locally and run it.
[REQUIRED]: In your css file, kindly remove the commented code.
[REQUIRED]: Remove the commented code in html line 38
[REQUIRED]: Here is a good way to specify paths.
Good: `<link rel="stylesheet" href="./style.css">`
Bad: `<link rel="stylesheet" href="style.css">`
[REQUIRED]: The new york times logo in the navbar should be a link to the same page
[REQUIRED]: Add the aside section to the project.
[REQUIRED]: Add some padding to the section under the image labelled "the bicep2 telescope".
[REQUIRED]: The section starting from the theory of inflation should use more font size and more precise styling. Kindly look at the design provided by Microverse.

Kindly make the required changes and ask for another review.👍👌
You got it.
Happy coding💻
[REQUIRED]: Kindly check the original page to make the footer look the same.

Chike1990 committed:
Updates the readme file and adds project screenshot

Chike1990 committed:
Removes commented codes on the CSS file;
Correctly specifies CSS file path in the index file

Chike1990 committed:
Initial work on aside

Chike1990 committed:
Adds footer

Chike1990 committed:
Completes work on footer as indicated by the reviewer

Chike1990 committed:
Fix styling issues on linter

Chike1990 committed:
Fixes linters

voscarmv reviewed (changes requested):
# Impressive work. 💻 👀 ✨ 

- Good job applying the changes requested by the previous TSE.

## However, there are some changes required 👀 🧰 🔨 🔧 
- Please use a font that resembles The New York Times' logo more closely.
- Please use some proper spacing between your paragraphs to make them resemble the original article's paragraph.
- The screenshot you took for your README file appears stretched. Please take one which does not distort the appearance of your page.
- The Live Demo link in your README file does not link to the page. Please use a link that takes the user to the page you created, either using https://rawcdn.githack.com/ or github pages.
### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

Chike1990 committed:
Adds proper spacing between paragraphs as on original article

Chike1990 committed:
Updates ReadMe file

Chike1990 committed:
Adds live Project link to README file

Chike1990 committed:
Add a font that resembles The New York Times' logo more closely

Chike1990 committed:
Updates READme file

Chike1990 committed:
Updates ReadMe file

Chike1990 committed:
Updates ReadMe file

Chike1990 committed:
Merge branch 'website-branch' of https://github.com/Chike1990/nyt_project into website-branch

Chike1990 committed:
Updates ReadMe

rindrajosia reviewed (changes requested):
Hello
Status: Changes Required recycle

Great job so far! :clap:
Let's cover a few details together to make your project better :blush: :arrow_double_down:

REQUIRED:
- [ ] Make sure there are no errors or warnings on the HTML validator. [Use this one.](https://validator.w3.org/nu)
- [ ] Add this section in your page
![1](https://user-images.githubusercontent.com/58552748/108572704-f0723c00-7323-11eb-8423-2cb51f30546c.png)

- [ ] Using `<br>` is bad when you try to add more space between 2 paragraphs. You can create class in your css file with margin bottom or margin top.
- [ ] Try to use the same font size and font weight as the original web page.
- [ ] I don't see the `Editors’ Picks` in your web page when I open it on my browser.
- [ ] The navigation bar and footer should stick to the top/bottom part of the screen.
- [ ] Elements aligned with a combination of all three techniques (float, flex and grid). As I can see, you are just using flex. Add float and grid.
- [ ] **Your pull request summary must be descriptive. Add a summary to your pull request.**
- [ ] Please, configure `linter` in your project. these files are missing:
![2](https://user-images.githubusercontent.com/58552748/108573707-5e1f6780-7326-11eb-8297-c23810efa8b0.png)
Follow instructions in this repo [https://github.com/microverseinc/linters-config/tree/master/html-css](https://github.com/microverseinc/linters-config/tree/master/html-css)

That's it, you can submit another code review request after making these changes
With all my respect :pray:

Rindra Josia

Chike1990 commented:
Alright, Rindra

Thank you,

Chibuike Uzoechina

On Sat, Feb 20, 2021 at 12:52 AM Rakotonarivo Rindra Josia <
notifications@github.com> wrote:

> *@rindrajosia* requested changes on this pull request.
>
> Hello
> Status: Changes Required recycle
>
> Great job so far! 👏
> Let's cover a few details together to make your project better 😊 ⏬
>
> REQUIRED:
>
>    -
>
>    Make sure there are no errors or warnings on the HTML validator. Use
>    this one. <https://validator.w3.org/nu>
>    -
>
>    Add this section in your page
>    [image: 1]
>    <https://user-images.githubusercontent.com/58552748/108572704-f0723c00-7323-11eb-8423-2cb51f30546c.png>
>    -
>
>    Using <br> is bad when you try to add more space between 2 paragraphs.
>    You can create class in your css file with margin bottom or margin top.
>    -
>
>    Try to use the same font size and font weight as the original web page.
>    -
>
>    I don't see the Editors’ Picks in your web page when I open it on my
>    browser.
>    -
>
>    The navigation bar and footer should stick to the top/bottom part of
>    the screen.
>    -
>
>    Elements aligned with a combination of all three techniques (float,
>    flex and grid). As I can see, you are just using flex. Add float and grid.
>    -
>
>    *Your pull request summary must be descriptive. Add a summary to your
>    pull request.*
>    -
>
>    Please, configure linter in your project. these files are missing:
>    [image: 2]
>    <https://user-images.githubusercontent.com/58552748/108573707-5e1f6780-7326-11eb-8297-c23810efa8b0.png>
>    Follow instructions in this repo
>    https://github.com/microverseinc/linters-config/tree/master/html-css
>
> That's it, you can submit another code review request after making these
> changes
> With all my respect 🙏
>
> Rindra Josia
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Chike1990/nyt_project/pull/1#pullrequestreview-594656270>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AO2A5OAQ2NSZWESWQAMEAP3S732SPANCNFSM4XXIFTBA>
> .
>

Chike1990 committed:
Final corrections on index and style files

Chike1990 committed:
Fixes issues identified on linters

Chike1990 committed:
Fixes linter issues

Chike1990 committed:
Last corrections on linters

Chike1990 committed:
Corrects font issues style file

Chike1990 commented:
The third reviewer has just pointed out missing elements on the web page I created. They include the errors with my HTML validator, missing aside bar, some bad practice -the use of <br> and absence of sticky nav bar. I have corrected those. Thanks for your time.

voscarmv reviewed (changes requested):
# Impressive work. 💻 👀 ✨ 

- Your project is very complete.

## However, there are some changes required 👀 🧰 🔨 🔧 

- The only mistake I could find is that you did not add a proper Pull Request description. Please take some time to make a list of the new concepts you learned working on this project, and add that to the PR description.
![image](https://user-images.githubusercontent.com/2739245/108608990-5694a880-7390-11eb-9b64-95b66b023e40.png)


### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

Chike1990 commented:
I cloned an article page of the New York Times applying the concepts I learnt over the week. Specifically,
I first created a repository on Github, created a branch from it, added ReadMe file (added a demo), added linter checks, index.html file, and style.css file.
The structure and content of the New York Times article page were laid out in HTML- using HTML 5 semantic tags; the images were downloaded from the NYT page and were included in the cloned web page; these contents were style using CSS - and by applying concepts like positioning, flex, float, etc.
In other words, this page includes a navbar, an aside, main, sections, articles, divs, footer, etc. Prior to sending a pull request, linters were checked for errors in the style and index files and about five code reviewers reviewed this code and made recommendations that have been included.

MarilenaRoque commented:
## Hello @Chike1990! :rocket:

 Nice implementation of the required changes! 
**Hard work always pays off.** :muscle:

# Project Approved! 🏅 

**Just a Final Hint**: You can edit your pull request message by clicking on the dots in the header.

Happy Coding! 💻 

**_Marilena Roque
Full-Stack Developer
Code Reviewer at Microverse Inc.
[Portfolio](https://mariroqueproject.com/) | [Linkedin](https://www.linkedin.com/in/roquemarilena/) 
marilenas@id.uf.br_**

MarilenaRoque reviewed (approved):
# Approved! :tada:

Chike1990 commented:
Thanks a lot.

On Sun, Feb 21, 2021, 4:03 AM Mari Roque <notifications@github.com> wrote:

> *@MarilenaRoque* approved this pull request.
> Approved! 🎉
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Chike1990/nyt_project/pull/1#pullrequestreview-594802266>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AO2A5OB7AUGDD5VGASMLFPDTABZXLANCNFSM4XXIFTBA>
> .
>
